pkg/costmodel/cluster.go

```diff
@@ -1436,10 +1441,6 @@ func pvCosts(diskMap map[DiskIdentifier]*Disk, resolution time.Duration, resActi
 		}
 
 		diskMap[key].Cost = cost * (diskMap[key].Bytes / 1024 / 1024 / 1024) * (diskMap[key].Minutes / 60)
-		providerID, _ := result.GetString("provider_id") // just put the providerID set up here, it's the simplest query.
-		if providerID != "" {
-			diskMap[key].ProviderID = provider.ParsePVID(providerID)
-		}
```

Comment:
I think that this is going to break reconciliation. Sorry for taking so long on this review... I wanted to find the proof first.

Reply:
Here's my reasoning: Today, reconciliation of disks works by fetching CUR data, including so-called "provider ID" and cost, and matching that data with Asset data on this field. If today the match works, and I have no reason to believe it doesn't, then we cannot change the value here without also changing it in reconciliation. (Or else, this change works by virtue of the fact that it does not change the value of provider ID, in which case it is moot.)

Does that make sense? We could equivalently try to run this, and see if the provider ID values change -- and if they do, then we can check if reconciliation breaks.

Reply:
You might check with @Sean-Holcomb on cloud IDs and which fields are and are not available in the CUR, too. He may know more than I do here.